conditions/README.md

```diff
@@ -0,0 +1,341 @@
+                <id root="11c13385-04a4-44d8-ba4a-B6F137A9BD47"/>
+            </statementRef>
+        </component>
+        <conditionNamed typeCode="NAME" inversionInd="true">
+            <namedStatementRef classCode="OBS" moodCode="EVN">
+                <id root="D122054B-9740-44F3-9592-604F9352C9BA"/>
+            </namedStatementRef>
+        </conditionNamed>
+        <Participant typeCode="PRF" contextControlCode="OP">
+            <agentRef classCode="AGNT">
+                <id root="4835D5BA-B9BF-4D49-A4DC-79C536328A3F"/>
+            </agentRef>
+        </Participant>
+    </LinkSet>
+</component>
+<component typeCode="COMP">
+<ObservationStatement classCode="OBS" moodCode="EVN">
+    <id root="D122054B-9740-44F3-9592-604F9352C9BA"/>
+    <code code="152306018" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Allergy to peanuts">
+    </code>
+    <statusCode code="COMPLETE"/>
+    <effectiveTime>
+        <low value="20100113"/>
+        <high value="20100323"/>
+    </effectiveTime>
+    <availabilityTime value="20100113140445"/>
+    <pertinentInformation typeCode="PERT">
+        <sequenceNumber value="+1"/>
+        <pertinentAnnotation classCode="OBS" moodCode="EVN">
+            <text>Problem Info: Related Problem: Child of Allergy to aspirin Problem Notes: This is a peanut allergy
+                (Evolved into H/O: aspirin allergy 13-Jan-2010)
+            </text>
+        </pertinentAnnotation>
+    </pertinentInformation>
+</ObservationStatement>
+</component>
+```
+
+</details>
+
+4. Where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ProblemSignificance-1"`
+5. For each `extension[index].value`, where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedClinicalContent-1"`
+   or `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1"`  
+   and:`extension[index].value` does not reference:
+- a [List](../list/README.md)
+- an [Encounter](../encounters/README.md)
+  or:
+- a suppressed [MedicationRequest](../medication%20requests/README.md) (with `MedicationRequest.status` == `"stopped"` and `MedicationRequest.intent` == `"order"`)
+
+6. where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1"`
+7. where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedProblemHeader-1"`
+
+#### CD for transformed actual problem header
+
+| Mapped to (XML HL7 CD) | Mapped from (JSON FHIR / other source )         |
+|------------------------|-------------------------------------------------|
+| code \[@code]          | fixed value = `"55607006"`                      |
+| code \[@codeSystem]    | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15` |
+| code \[@displayName]   | fixed value = `"Problem"`                       |
```

Comment:
Are we missing a row for the `code / originalText` ?

Reply:
Yes I'll add it